services/crates/crates-license.tester.js

```diff
@@ -16,4 +11,12 @@ t.create('license (with version)')
 
 t.create('license (not found)')
   .get('/not-a-real-package.json')
-  .expectBadge({ label: 'crates.io', message: 'not found' })
+  .expectBadge({ label: 'license', message: 'not found' })
+
+t.create('license (null licenses in history)')
+  .get('/stun.json')
```

Comment:
This is an example of such a specific condition, lets link to https://github.com/badges/shields/issues/7073 in a comment here for future reference